aws-amplify/docs
# Use canonical url from meta if it exists

#### Description of changes:
- Add logic to use the `canonicalUrl` property from meta as the canonical Url if it exists

Staging site: https://custom-canonical-url.d1ywzrxfkb9wgg.amplifyapp.com/

To test:
1. Go to https://custom-canonical-url.d1ywzrxfkb9wgg.amplifyapp.com/swift/build-a-backend/more-features/datastore/data-usage-policy
2. Check the page source and verify that the canonical url is set to https://docs.amplify.aws/swift/build-a-backend/auth/data-usage-policy

To test canonical-urls.ts changes:
1. Go to https://custom-canonical-url.d1ywzrxfkb9wgg.amplifyapp.com/swift/tools/cli/project/permissions-boundary/
2. Verify that the canonical url is set to https://docs.amplify.aws/javascript/tools/cli/project/permissions-boundary/

#### Related GitHub issue #, if available:

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?

Which platform(s) are affected by this PR (if applicable)?

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks




      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._
